topics/shell/directories_check.md

```diff
@@ -0,0 +1,5 @@
+## Directories Check
+
+### Objectives
+
+1. write a script to Check if directory exists.
```

Comment:
```suggestion
1. Write a script to Check if directory exists.
```

Reply:
Thank you @andrzejsydor just made the correction.. 
I think it's OK, for a approval now.
Thank you..